scripts/bootstrap-plugin

```diff
@@ -21,8 +21,6 @@ if [ -d $pluginname ]; then
   err "There is already a plugin with that name. Cowardly refusing to overwrite it."
 fi
 
-buildConfigFilename=${pluginname}/build.config.js
-
 mvn archetype:generate -DarchetypeGroupId=org.graylog -DarchetypeArtifactId=graylog-plugin-archetype -DartifactId=$pluginname
 
-sed 's,\(../graylog2-server/graylog2-web-interface\),../\1/,' < ${buildConfigFilename}.sample > ${buildConfigFilename}
+echo ${pluginname}/ >> ${prefix}/.gitignore
```

Comment:
Hmm, I don't get where the difference is between putting all repos into `graylog-project/<repo>` or into `graylog-project/modules/<repo>` from a `build.conf.js` perspective. Wouldn't the move into `graylog-project/<repo>` require a change to `build.conf.js` aswell?  Can you elaborate?